### PR TITLE
ad9739a.c & dac_data_manager.c: Change name from "hpc" to "lpc"

### PR DESCRIPTION
**Pull request by IuliaCMoldovan**
Linked to [ADI Linux PR 2125](https://github.com/analogdevicesinc/linux/pull/2125).

### CODE REVIEW

**Comment by dNechita** [comment]
Would it make sense to make this backwards compatible? Meaning that for whoever uses some old designs and still has -lpc suffix this plugin will still work. Or is the design so old that no one will use?

File: plugins/dac_data_manager.c [hunk 1]
```diff
@@ -841,7 +841,11 @@ static GtkWidget *gui_dds_mode_chooser_create(struct dds_tx *tx)
 	GtkWidget *box;
 	GtkWidget *dds_mode_lbl;
 	GtkComboBoxText *dds_mode;
-	bool no_buffer_support = !strcmp(tx->parent->name, "axi-ad9739a-hpc");
+	bool no_buffer_support=false;
+	if(!strcmp(tx->parent->name,"axi-ad9739a-lpc"))
+	  no_buffer_support = !strcmp(tx->parent->name, "axi-ad9739a-lpc");
```
Comment (dNechita) [inline plugins/dac_data_manager.c:846]: Well, you can directly set no_buffer_support to true. You already did the comparison in the if statement.
You can even go further and do something like:
if (name is axi-ad9739a-lpc or is axi-ad9739a-hpc) then set no_buffer_suport to true. Which I believe is more readable.

Comment (cristina-suteu) [reply plugins/dac_data_manager.c]: Fixed

**Comment by cristina-suteu** [comment]
This PR is pending hardware testing

**Review by dNechita — APPROVED** [review]

**Pull request merged**